src/android/InAppBrowser.java

```diff
@@ -1522,6 +1632,8 @@ public void onPageStarted(WebView view, String url, Bitmap favicon) {
         public void onPageFinished(WebView view, String url) {
             super.onPageFinished(view, url);
 
+            pageTitleTextView.setText(view.getTitle());
```

Comment:
Should this be conditioned on navigation bar being present?

Reply:
Yep, I'll add the conditional!